_includes/version_warning.html

```diff
@@ -1,5 +1,5 @@
 {% if page.version != site.data.versions.first[0] %}
   <div class="alert alert-warning" role="alert">
-    <strong>Warning:</strong> You are viewing the {{ page.version }} docs. <a href="{{site.baseurl}}/latest">Click here to view the docs for the latest release.</a>
+	  <strong>Warning:</strong> You are viewing the {{ page.version }} docs. <a href="{{site.baseurl}}/latest/{{page.url | remove:page.version }}">Click here to view the docs for the latest release.</a>
```

Comment:
Please can you pull this change out into a separate PR and update the 404 page so that, if the document doesn't exist in /latest/ the user doesn't get confused?

Reply:
My bad ! this was a fix to another bug. Let me re-organise it to another branch and open a separate PR for the same. Realised after pushing.

Reply:
The fix to the other bug was pushed only today so you couldn't have spotted it earlier @fasaxc